### PR TITLE
admin user index, phan quyen

### PR DESCRIPTION
**Pull request by thanhnn-3239**
[](url)## Related Tickets
- [#50949](https://edu-redmine.sun-asterisk.vn/issues/50949)

## WHAT (optional)
- admin user index, phan quyen

## HOW
- admin user index, phan quyen

## WHY (optional)
- admin user index, phan quyen

## Evidence (Screenshot or Video)
Screenshot from 2022-07-19 16-09-47

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by thanhnn-3239** [comment]
ready

**Comment by thanhnn-3239** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -18,4 +18,12 @@ def destroy
 
     redirect_to root_path
   end
+
+  def redirect user
```
Comment (huongnt-2545) [inline app/controllers/sessions_controller.rb:24]: private method

File: app/models/user.rb [hunk 2]
```diff
@@ -17,6 +20,9 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  scope :asc_name, ->{order name: :asc}
+  scope :get_all, ->{select(GET_ALL).where(role: 1)}
```
Comment (huongnt-2545) [inline app/models/user.rb]: `where(role: 1)` -> chỗ này em dùng tính năng của enum để ấy ra đc user là customer
vd: `User.customer`

File: app/views/admin/books/_book.html.erb [hunk 3]
```diff
@@ -12,7 +12,7 @@
   <% end %>
   <td>
     <%= link_to edit_admin_book_path(book) do %>
-       <label class="btn btn-warning">
+        <label class="btn btn-warning">
```
Comment (huongnt-2545) [inline app/views/admin/books/_book.html.erb]: sai indent -> indent = 2 spaces

**Review by huongnt-2545 — COMMENTED** [review]
commented

**Comment by thanhnn-3239** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**